accounting/accounting_cli.py

```diff
@@ -82,7 +87,19 @@ def main():
 
     args = parser.parse_args()
 
-    conn = sqlite3.connect("FluxAccounting.db")
+    # try to open database file; will exit with -1 if database file not found
+    if args.path:
+        try:
+            conn = sqlite3.connect("file:" + args.path + "?mode=rw", uri=True)
+        except sqlite3.OperationalError:
+            print("Unable to open database file")
+            sys.exit(-1)
+    else:
+        try:
+            conn = sqlite3.connect("file:FluxAccounting.db?mode=rw", uri=True)
+        except sqlite3.OperationalError:
+            print("Unable to open database file")
+            sys.exit(-1)
```

Comment:
Nit, but you can eliminate some code duplication by leveraging a `path` variable. Something like:

```
        path = args.path if args.path else "FluxAccounting.db"
        try:
            conn = sqlite3.connect("file:" + path + "?mode=rw", uri=True)
        except sqlite3.OperationalError:
            print("Unable to open database file")
            sys.exit(-1)
```